### PR TITLE
Create Payment Screen Pt. 1

### PR DESCRIPTION
**Pull request by mhxynh**
**Ticket:** [Payment Screen](https://cs375-pos-group-project.atlassian.net/browse/ATS-20?atlOrigin=eyJpIjoiYzJlYzA2ZGI2MDY4NDExOWE0NWQwZWIyMmVkZmZjMzgiLCJwIjoiaiJ9)

- Created `pay.js`, which fetches `/api/items` to grab item prices for checkout calculations (WIP)
- Need to still work on HTML page

### CODE REVIEW

**Review by gitnamehere — COMMENTED** [review]
you can remove create.html and create.js as it was moved over to another directory

File: app/server.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const express = require("express");
 const { Pool } = require("pg");
 const env = require("../env.json");
+const { error } = require("console");
```
Comment (gitnamehere) [inline app/server.js:4]: this too

**Review by nhokzminh698 — APPROVED** [review]
Yuh!

**Comment by gitnamehere** [comment]
acutually I think you should try updating from main to sync up the changes

**Review by gitnamehere — APPROVED** [review]

**Pull request merged**